Tests/MongoDBVaporTests/MongoDBVaporTests.swift

```diff
@@ -1,6 +1,20 @@
 @testable import MongoDBVapor
+import Nimble
+import Vapor
 import XCTest
 
 final class MongoDBVaporTests: XCTestCase {
-    func testExample() {}
+    func testSetUpAndCleanUp() throws {
+        var env = try Environment.detect()
+        try LoggingSystem.bootstrap(from: &env)
+        let app = Application(env)
+
+        try app.configureMongoDB()
+        expect(app.mongoClient).toNot(beNil())
+        expect(try app.mongoClient.listDatabases().wait()).toNot(throwError())
+
+        try app.cleanupMongoDB()
+        expect(try app.mongoClient.listDatabases().wait()).to(throwError())
```

Comment:
Should we test that a specific error is thrown when trying to use a client that has already been closed? Like a `ChannelError` (NIO)?

Reply:
I think it's fine that we just test that some error occurs here. I don't think its necessarily that important that we test which error is thrown (at least not in this library). We could add such a test in the driver if need be.